aws-geo-location/src/main/java/com/amplifyframework/geo/location/AWSLocationGeoPlugin.kt

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.geo.location
+
+import android.content.Context
+import com.amazonaws.services.geo.AmazonLocationClient
+import com.amplifyframework.AmplifyException
+import com.amplifyframework.core.Consumer
+import com.amplifyframework.geo.GeoCategoryPlugin
+import com.amplifyframework.geo.GeoException
+import com.amplifyframework.geo.location.configuration.GeoConfiguration
+import com.amplifyframework.geo.models.MapStyle
+import org.json.JSONObject
+
+/**
+ * A plugin for the Geo category to interact with Amazon Location Service.
+ */
+class AWSLocationGeoPlugin(
+    private val userConfiguration: GeoConfiguration? = null // for programmatically overriding amplifyconfiguration.json
+): GeoCategoryPlugin<AmazonLocationClient?>() {
+    companion object {
+        private const val GEO_PLUGIN_KEY = "awsLocationGeoPlugin"
```

Comment:
For constants it looks like you have to use `@JvmField` annotation.

https://developer.android.com/kotlin/interop#companion_constants

Reply:
it's a private constant, so it won't be exposed

Reply:
Doh!!! 🤣